assembl/static2/js/app/components/administration/discussion/manageSectionsForm.jsx

```diff
@@ -35,4 +43,13 @@ const mapStateToProps = (state) => {
   };
 };
 
-export default connect(mapStateToProps)(ManageSectionsForm);
\ No newline at end of file
+const mapDispatchToProps = (dispatch) => {
+  return {
+    createSection: (nextOrder) => {
+      const newId = Math.round(Math.random() * -1000000).toString();
```

Comment:
It feels weird to find this line at this place. It is either not the right place (should be in a utility method), or probably just not needed? When you create a new element you should not have to worry about its id as it is not set yet and will be decided by the backend. For example why just not provide `-1`? It would not work if we try to create several elements?

Reply:
The id needs to be unique in the frontend, so we generate a random negative number that will be overriden by the postive id that the backend returns.

Reply:
yes, we need an id for the redux states, it ease the work. I agree that we could move it somewhere in `utils`